tests/phpunit/tests/REST_API/Autosaves_Controller.php

```diff
@@ -17,6 +17,7 @@
 
 namespace Google\Web_Stories\Tests\REST_API;
 
+use Google\Web_Stories\Story_Post_Type;
```

Comment:
Unrelated change?

Reply:
Tiny cleanup I happened to add when testing failing autosaves test. Kept it for readability reasons.